AT_EFD/docker-compose.yaml

```diff
@@ -1,103 +1,154 @@
                 environment:
                         - LSST_DDS_DOMAIN=${LSST_DDS_DOMAIN}
                         - LSST_EFD_HOST=${LSST_EFD_HOST}
                         - LSST_KAFKA_BROKERS=localhost
-                network_mode: "host"
+                networks:
+                        control-network:
+                                ipv4_address: "192.168.1.6"
                 volumes:
-                        - ${OSPL_CONFIG_PATH}:${OSPL_MOUNT_POINT}
                         - ./efd_writers.sh:/home/saluser/efd_writers.sh
                 entrypoint: ["/home/saluser/efd_writers.sh" , "ATMCS" , "efdwriter"]
         EFD_ATptg:
-                image: lsstts/efd_writers
+                image: lsstts/efd_writers:20190514
                 environment:
                         - LSST_DDS_DOMAIN=${LSST_DDS_DOMAIN}
                         - LSST_EFD_HOST=${LSST_EFD_HOST}
                         - LSST_KAFKA_BROKERS=localhost
-                network_mode: "host"
+                networks:
+                        control-network:
+                                ipv4_address: "192.168.1.7"
                 volumes:
-                        - ${OSPL_CONFIG_PATH}:${OSPL_MOUNT_POINT}
                         - ./efd_writers.sh:/home/saluser/efd_writers.sh
                 entrypoint: ["/home/saluser/efd_writers.sh" , "ATPtg" , "efdwriter"]
         EFD_ATDomeTrajectory:
-                image: lsstts/efd_writers
+                image: lsstts/efd_writers:20190514
                 environment:
                         - LSST_DDS_DOMAIN=${LSST_DDS_DOMAIN}
                         - LSST_EFD_HOST=${LSST_EFD_HOST}
                         - LSST_KAFKA_BROKERS=localhost
-                network_mode: "host"
+                networks:
+                        control-network:
+                                ipv4_address: "192.168.1.8"
                 volumes:
-                        - ${OSPL_CONFIG_PATH}:${OSPL_MOUNT_POINT}
                         - ./efd_writers.sh:/home/saluser/efd_writers.sh
                 entrypoint: ["/home/saluser/efd_writers.sh" , "ATDomeTrajectory" , "efdwriter"]
         EFD_ATPneumatics:
-                image: lsstts/efd_writers
+                image: lsstts/efd_writers:20190514
                 environment:
                         - LSST_DDS_DOMAIN=${LSST_DDS_DOMAIN}
                         - LSST_EFD_HOST=${LSST_EFD_HOST}
                         - LSST_KAFKA_BROKERS=localhost
-                network_mode: "host"
+                networks:
+                        control-network:
+                                ipv4_address: "192.168.1.9"
                 volumes:
                         - ${OSPL_CONFIG_PATH}:${OSPL_MOUNT_POINT}
                         - ./efd_writers.sh:/home/saluser/efd_writers.sh
                 entrypoint: ["/home/saluser/efd_writers.sh" , "ATPneumatics" , "efdwriter"]
         EFD_ATAOS:
-                image: lsstts/efd_writers
+                image: lsstts/efd_writers:20190514
```

Comment:
would some version tag be better -- I assume this is a date ?

Reply:
yes... I still need to work on a better tagging system. this is the build date of the container.